darts/models/forecasting/multivariate_forecasting_model_wrapper.py

```diff
@@ -0,0 +1,151 @@
+"""
+Regression ensemble model
+-------------------------
+
+An ensemble model which uses a regression model to compute the ensemble forecast.
+"""
+from typing import List, Optional, Tuple
+
+from darts.logging import get_logger
+from darts.models.forecasting.forecasting_model import (
+    FutureCovariatesLocalForecastingModel,
+    LocalForecastingModel,
+    TransferableFutureCovariatesLocalForecastingModel,
+)
+from darts.timeseries import TimeSeries, concatenate
+from darts.utils.utils import seq2series
+
+logger = get_logger(__name__)
+
+
+class MultivariateForecastingModelWrapper(FutureCovariatesLocalForecastingModel):
+    def _fit(self, series: TimeSeries, future_covariates: Optional[TimeSeries] = None):
+        super()._fit(series, future_covariates)
+        self._trained_models = []
+
+        series = seq2series(series)
+        components = self._split_multivariate(series)
+        if self.supports_future_covariates:
+            self._trained_models = [
+                self.base_model.untrained_model().fit(c, future_covariates)
+                for c in components
+            ]
+        else:
+            self._trained_models = [
+                self.base_model.untrained_model().fit(c) for c in components
+            ]
+
+        return self
+
+    def predict(
+        self,
+        n: int,
+        series: Optional[TimeSeries] = None,
+        future_covariates: Optional[TimeSeries] = None,
+        num_samples: int = 1,
+        **kwargs,
+    ) -> TimeSeries:
+        # we override `predict()` to pass a non-None `series`, so that historic_future_covariates
+        # will be passed to `_predict()` (some future covariates local models require it ex. Kalman)
+        return self._predict(n, future_covariates, num_samples, **kwargs)
+
+    def _predict(
+        self,
+        n: int,
+        future_covariates: Optional[TimeSeries] = None,
+        num_samples: int = 1,
+        verbose: bool = False,
+        **kwargs,
+    ) -> TimeSeries:
+        predictions = [
+            model.predict(n=n, future_covariates=future_covariates)
+            if isinstance(model, FutureCovariatesLocalForecastingModel)
+            else model.predict(n=n)
+            for model in self._trained_models
+        ]
+
+        multivariate_series = concatenate(predictions, axis=1)
+        return multivariate_series
+
+    def __init__(self, model: LocalForecastingModel):
+        """
+        Wrapper around LocalForecastingModel allowing it to act like a GlobalForecastingModel
+
+        A copy of the provided model will be trained on each of the components of the provided series separately.
+        The model doesn't use series supplied during predict() and instead predicts on the series it trained on.
+
+        Parameters
+        ----------
+        model
+            Model used to predict individual components
+        """
+        super().__init__()
+
+        self.base_model: LocalForecastingModel = model
```

Comment:
I would check that the model passed is an instance of `LocalForecastingModel`. And call the attribute `model` to make it more intuitive.